ui/src/components/VaultSummary.jsx

```diff
@@ -74,6 +75,16 @@ const calcRatio = (priceRate, newLock, newBorrow) => {
   return makeRatioFromAmounts(lockPrice, newBorrow);
 };
 
+/**
+ * @typedef {Object} Props
+ * @property {VaultData} vault
+ * @property {Object} brandToInfo
+ * @property {string} id
+ */
```

Comment:
just fyi, it might be less work to type:
```suggestion
/**
 * @typedef {{
 * vault: VaultData,
 * brandIToInfo: Object,
 * id: string,
 * }} Props
 */
```
That won't get auto-formatted until https://github.com/Agoric/agoric-sdk/pull/4903

as an aside, is there a better type for `brandToInfo`?

Reply:
Ah yes much better. And yes just fixed that, good catch.